webauthn/webauthn.py

```diff
@@ -103,6 +105,13 @@ def __init__(self, challenge, rp_name, rp_id, user_id, username,
                              ', '.join(self._attestation_forms))
         self.attestation = attestation
 
+        if userVerification != None:
```

Comment:
If the `userVerification` default is one of `self._userVerification`, then we don't need this check. Or were you thinking that you didn't want the JSON payload to be different? If that's the case, can we at least say `if userVerification is not None` instead?

Reply:
I was simply coding this part like the check for the attestation, in line 102 in this PR.
I think that uniformity in logic is more important than code cleverness.

On the other hand, it make the api more flexible (see comment https://github.com/duo-labs/py_webauthn/pull/43#discussion_r312464220 also).

Reply:
Sounds good to me. Can you just change it to `if user_verification is not None:`?

Reply:
yeap